src/lib.rs

```diff
@@ -205,6 +205,28 @@ pub struct Buffer<'a, D, W> {
 }
 
 impl<D: HasDisplayHandle, W: HasWindowHandle> Buffer<'_, D, W> {
+    /// The amount of pixels wide the buffer is.
+    pub fn width(&self) -> usize {
```

Comment:
I went with `usize` here, since it was just much more usable in examples.

I'd like to change `resize` to follow this direction as well, see:
https://github.com/rust-windowing/softbuffer/issues/238

Reply:
I would prefer `NonZeroU32` to accurately reflect the type information.

Reply:
I went with `NonZeroU32`, it's not as bad as I feared. I'll push forwards with just using `u32` in the future.